beacon_node/http_api/tests/tests.rs

```diff
@@ -1933,19 +1933,24 @@ impl ApiTester {
     pub async fn test_get_beacon_light_client_bootstrap(self) -> Self {
         let block_id = BlockId(CoreBlockId::Finalized);
         let (block_root, _, _) = block_id.root(&self.chain).unwrap();
-        let (block, _, _) = block_id.full_block(&self.chain).await.unwrap();
 
         let result = match self
             .client
             .get_light_client_bootstrap::<E>(block_root)
             .await
         {
-            Ok(result) => result.unwrap().data,
+            Ok(result) => result,
             Err(e) => panic!("query failed incorrectly: {e:?}"),
         };
 
-        let expected = block.slot();
-        assert_eq!(result.get_slot(), expected);
+        assert!(result.is_none());
```

Comment:
why do we expect this test to fail now? because the server isn't enabled in these tests?

Reply:
yeah exactly. it might be beneficial to enable the light client server in http tests to get full test coverage for the `light_client_bootstrap` and `light_client_updates` endpoint

Reply:
it should be fairly trivial to enable the light client server for these tests. I'll just go ahead and do that

Reply:
e141c80